src/content/docs/en/reference/api-reference.mdx

```diff
@@ -778,10 +826,79 @@ const draftBlogPosts = await getCollection('blog', ({ data }) => {
 
 [See the `Content Collection` guide](/en/guides/content-collections/#querying-collections) for example usage.
 
+### `getEntry()`
+
+<p>
+  <Since v="2.5.0" />
+</p>
+
+**Types:**
+
+- `(collection: string, contentSlugOrDataId: string) => CollectionEntry<collection>`
+- `({ collection: string, id: string }) => CollectionEntry<collection>`
+- `({ collection: string, slug: string }) => CollectionEntry<collection>`
+
+`getEntry()` is a function that retrieves a single collection entry by collection name and either the entry `id` (for JSON or YAML collections) or entry `slug` (for Markdown or MDX collections).
```

Comment:
Quick question: reading the content-collections page, it made it seem more like `id` existed for both, but `slug` is only available for `type: content`. This makes it sound like you choose `id` for data, and slug for content.

Also, this says Markdown or MDX but doesn't mention Markdoc. So, we should maybe future-proof (and make things shorter!) by refering to the `type` type (right now, `data` and `content`) when we need to differentiate, vs file type?

Reply:
Correct, `id` _does_ exist for both. But in practice, we've found that users rarely use the _Content_ `id` since it isn't URL-friendly. This means it isn't used for `getStaticPaths()`, isn't used for SSR lookups, isn't used for... much of anything. The goal is to treat `slug` as the de-facto identifier now that we know how people use it, and come up with a much nicer solution come v3!